internal/adapters/cloudformation/aws/rds/cluster.go

```diff
@@ -28,6 +28,7 @@ func getClusters(ctx parser.FileContext) (clusters map[string]rds.Cluster) {
 			Engine:               defsecTypes.StringDefault(rds.EngineAurora, clusterResource.Metadata()),
 			LatestRestorableTime: defsecTypes.TimeUnresolvable(clusterResource.Metadata()),
 			DeletionProtection:   defsecTypes.BoolDefault(false, clusterResource.Metadata()),
+			SkipFinalSnapshot:    defsecTypes.BoolDefault(false, clusterResource.Metadata()),
```

Comment:
It's the same as https://github.com/aquasecurity/defsec/pull/1449/files#r1319740047 .